### PR TITLE
Fix path

### LINKED ISSUE #86: Wrong path to matomo script
### Description
Update ` src: '/lisk-documentation/js/matomo.js',` to ` src: '/js/matomo.js',

### Location
`docusaurus.config.js`

### Additional Information
Old path was only valid for github pages

### PR DESCRIPTION
**Pull request by Tschakki**
### What was the problem?

This PR resolves #86

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ const config = {
 
   scripts: [
     {
-      src: '/lisk-documentation/js/matomo.js',
+      src: '/js/matomo.js',
```
Comment (shuse2) [inline docusaurus.config.js]: probably you need to use append `process.env.LISK_DOC_BASE_URL`?

Comment (Tschakki) [reply docusaurus.config.js]: Hmm, but we do not really want Matomo to track on staging anyway

Comment (shuse2) [reply docusaurus.config.js]: i guess that's true, but it seems it's wrong 🤔 also number in the staging probably doesn't affect much anyway

Comment (Tschakki) [reply docusaurus.config.js]: > but it seems it's wrong
what is wrong exactly?

```suggestion
      src: process.env.LISK_DOC_BASE_URL + '/js/matomo.js',
```

you mean update it like this? Imho we should only do that if tracking on github pages is blocked. No reason to pollute the tracking results unnecessary. I think it should be blocked by default, but I saw something got tracked on matomo anyway. So if we make this change we should double check. 

But imho we can just leave as is, and there should be no problem, as it works on production, but not on staging, like intended.

Comment (shuse2) [reply docusaurus.config.js]: yea I meant something like that. to do it correctly, we can ignore from certain URL on the matomo side.

Comment (Tschakki) [reply docusaurus.config.js]: @shuse2 made the change, please check!

**Review by TalhaMaliktz — APPROVED** [review]
LGTM

File: docusaurus.config.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ const config = {
 
   scripts: [
     {
-      src: '/js/matomo.js',
+      src: process.env.LISK_DOC_URL ?? '/' + 'js/matomo.js',
```
Comment (shuse2) [inline docusaurus.config.js:41]: isn't it suppose to be something like
```suggestion
      src: `${process.env.LISK_DOC_URL ?? ''}/js/matomo.js`
```

Comment (Tschakki) [reply docusaurus.config.js:41]: isn't this just an alternative way to concatenate strings?

Comment (shuse2) [reply docusaurus.config.js:41]: Ah actually I miss read what it's doing. All good

**Review by shuse2 — APPROVED** [review]

**Pull request merged**